playbooks/install_stack.yaml

```diff
@@ -228,7 +228,7 @@
         standalone_extra_config: "{{ standalone_extra_config | combine(manila_extra_config) }}"
       vars:
         manila_extra_config:
-          ganesha_vip: "{{ control_plane_ip }}"
+          ganesha_vip: "{{ public_api }}"
```

Comment:
I understand this is the fastest path to get manila tested in CI, however I'm not a big fan of exposing manila shares on the public API + IP filtering and would very much prefer we keep it on an internal network and deploy OCP computes with additional network as this will likely be closer to what customers will do.